bin/check-ports.rb

```diff
@@ -86,12 +86,19 @@ def run
         port
       end
     end
+
+    hosts = config[:hosts].split(',').flat_map do |host|
+      host
+    end
+
     okarray = []
-    ports.each do |port|
-      okarray << 'ok' if check_port port
+    hosts.each do |host|
+      ports.each do |port|
+        okarray << 'ok' if check_port port, host
```

Comment:
how about `ok_hosts` you should use parentheses around parameters in a function unless it is an internal DSL such as the options are.

Reply:
added parentheses